README.md

```diff
@@ -48,5 +48,6 @@ Main Known issue
 
 ## Known Working mods that add Peripherals
 * Please let me know of other mods that work with this one
-	* Better End
-	* Better Nether
+	* Better End  adds chests
```

Comment:
Please have the "features" on a second line like so
```md
* Better End
   * Adds Chests
   ```

Reply:
ok